sdk/core/azure-core/src/main/java/com/azure/android/core/util/function/BiConsumer.java

```diff
@@ -0,0 +1,8 @@
+package com.azure.android.core.util.function;
+
+/**
+ * Simple interface make code compatible with Java 7
+ */
+public interface BiConsumer<Input1, Input2> {
```

Comment:
If we end up taking my suggestion above about de-abstracting the Response Validation interceptor, we can get rid of this class.